Python/Product/Cookiecutter/Cookiecutter.csproj

```diff
@@ -35,6 +35,10 @@
     <AssemblyName>Microsoft.CookiecutterTools</AssemblyName>
     <UseVSSDK>true</UseVSSDK>
     <CreateVsixContainer>true</CreateVsixContainer>
+    <StartAction>Program</StartAction>
```

Comment:
Yep. We want it to start with VS by default on F5, and not force us to reconfigure it manually each time.